app/controllers/result_controller.rb

```diff
@@ -1,4 +1,3 @@
 class ResultController < ApplicationController
-  def index
-  end
+  def index; end
```

Comment:
I did have it like that originally but the linter complained saying it should be on one